packages/libraries/client/src/apollo.ts

```diff
@@ -39,7 +39,7 @@ export function hiveApollo(clientOrOptions: HiveClient | HivePluginOptions): Apo
         },
       });
 
-  hive.info();
+  void hive.info();
```

Comment:
I think in this line we don't need to await, in cases like this `void` operator can be used [more info on the docs](https://github.com/typescript-eslint/typescript-eslint/blob/main/packages/eslint-plugin/docs/rules/no-floating-promises.md#ignorevoid)